documentation/get-started/domain-map.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: sub-navigation
+order: 30
+title: Domain map
+description: Set of statements about how domain entities relate to each other.
+---
+
+## Direct relationships
+- A case is for a single case type
+- A case type is owned by a single business unit
+- A business unit can own multiple case types
+- A case consists of multiple stages
```

Comment:
I think MTS case type has only a single stage.

Reply:
I've not come across MTS, but multiple means 'one or more', so I think it's OK?